Header header
TerrainCell[] cell
float32 plane_size
float32 height_size